lib/gracenote/helper.rb

```diff
@@ -0,0 +1,38 @@
+module Gracenote
+  module Helper
+    def wrap_array(object)
+      case object
+      when NilClass
+        []
+      when Array
+        object
+      else
+        [object]
+      end
+    end
+
+    def recursive_downcase_keys(hash, numerify_values: false)
+      hash.each_with_object({}) do |(k,v), memo|
+        memo[k.downcase] = case v
+        when Hash
+          recursive_downcase_keys(v)
+        when Array
```

Comment:
__[rubocop]__
* Indent `when` as deep as `case`. [:ref](https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)